tc/sgx/trusted_worker_manager/enclave/work_order_data_handler.cpp

```diff
@@ -67,10 +65,10 @@ namespace tcf {
 
         input_data_hash = GetJsonStr(object, "dataHash");
         if (!encrypted_input_data.empty()) {
-            DecryptInputData(encrypted_input_data, iv);
-
+            DecryptInputData(data_encryption_key, iv, encrypted_input_data);
```

Comment:
if encrypted_input_data is null or data_encryption_key  is null then 
workorder_data.decrypted_data = encrypted_input_data;

Reply:
Ok